src/network/networkctl.c

```diff
@@ -792,102 +847,264 @@ static int link_status_one(
         (void) sd_network_link_get_carrier_bound_to(info->ifindex, &carrier_bound_to);
         (void) sd_network_link_get_carrier_bound_by(info->ifindex, &carrier_bound_by);
 
-        printf("%s%s%s %i: %s\n", on_color_operational, special_glyph(SPECIAL_GLYPH_BLACK_CIRCLE), off_color_operational, info->ifindex, info->name);
-
-        printf("            Link File: %s\n"
-               "         Network File: %s\n"
-               "                 Type: %s\n"
-               "                State: %s%s%s (%s%s%s)\n",
-               strna(link),
-               strna(network),
-               strna(t),
-               on_color_operational, strna(operational_state), off_color_operational,
-               on_color_setup, strna(setup_state), off_color_setup);
-
-        if (path)
-                printf("                 Path: %s\n", path);
-        if (driver)
-                printf("               Driver: %s\n", driver);
-        if (vendor)
-                printf("               Vendor: %s\n", vendor);
-        if (model)
-                printf("                Model: %s\n", model);
+        table = table_new("DOT", "KEY", "VALUE");
+        if (!table)
+                return -ENOMEM;
+
+        table_set_header(table, false);
+
+        r = table_add_cell(table, &cell, TABLE_STRING, special_glyph(SPECIAL_GLYPH_BLACK_CIRCLE));
+        if (r < 0)
+                return r;
+        assert_se(table_set_ellipsize_percent(table, cell, 0) >= 0);
+        assert_se(table_set_color(table, cell, on_color_operational) >= 0);
```

Comment:
as above, no assert_se(), this may fail due to oom... in fact all of these calls might